test/runnable/test22945.d

```diff
@@ -0,0 +1,40 @@
+module test.runnable.test22945 ;
```

Comment:
No need to add module name

Reply:
Oops, that was the vscode plugging sneaking in when I renamed the file.

Reply:
done